Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:12-slim
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Copies your code file from your action repository to the filesystem path `/` of the container
+COPY entrypoint.sh /entrypoint.sh
+
+# Copy all trivy report data
+COPY *.json /data/
+
+# Mount data volume
+VOLUME /data
+
+# Install required packages
+RUN apt-get update && \
+    apt-get install -y tar ca-certificates gnupg curl --no-install-recommends && \
+    # Import Docker GPG key
+    install -m 0755 -d /etc/apt/keyrings && \
```

Comment:
can we move these to the top so we avoid doing an extra apt-get update

Reply:
I'm not sure, since we need "ca-certificates gnupg curl" packages to import docker gpg key which is used by the repo. I tried it a couple ways without using another apt-get update but it doesn't compile

Reply:
I see, makes sense